### PR TITLE
Update apps-selective-wipe.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the line about selective data to reduce ambiguity

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by PRMerger-2** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/memdocs/issues/1960

File: memdocs/intune/apps/apps-selective-wipe.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ When a device is lost or stolen, or if the employee leaves your company, you wan
...
 
-To selectively remove company app data, create a wipe request by using the steps in this topic. After the request is finished, the next time the app runs on the device, company data is removed from the app. In addition to creating a wipe request, you can configure a selective wipe of your organization's data as a new action when the conditions of Application Protection Policies (APP) Access settings are not met. This feature helps you automatically protect and remove sensitive organization data from applications based on pre-configured criteria.
+To selectively remove company app data, create a wipe request by using the steps in this topic. After the request is finished, the next time the app runs on the device, company data is removed from the app. In addition, You can also configure a selective wipe of your Company data as a new action when the conditions of Application Protection Policies (APP) Access settings are not met. This feature helps you automatically protect and remove sensitive Company data from applications based on pre-configured criteria.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/apps-selective-wipe.md:43]:
```suggestion
To selectively remove company app data, create a wipe request by using the steps in this topic. After the request is finished, the next time the app runs on the device, company data is removed from the app. In addition, you can also configure a selective wipe of your company data as a new action when the conditions of Application Protection Policies (APP) Access settings are not met. This feature helps you automatically protect and remove sensitive company data from applications based on pre-configured criteria.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by PRMerger9** [comment]
@msbemba : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Erikre Please review and merge this PR. Thanks

**Pull request merged**